src/firefly/js/drawingLayers/CatalogUI.jsx

```diff
@@ -260,3 +263,44 @@ CatalogUI.propTypes= {
 };
 
 
+export function OptionalHighlight({viewerId, tbl_id,highlightRow, originalCurrentRow, tableRequest,renderTime}) {
+    const [visible,setVisible]= useState(true);
+    const currentRow= useStoreConnector(() => getTblById(tbl_id)?.highlightedRow);
+
+
+    const sx= (theme) => ({
+        alignItems:'center',
+        borderRadius: '5px',
+        overflow:'hidden',
+        border: '2px solid rgba(0,0,0,.1)',
+        borderColor: theme.vars.palette.warning.outlinedColor,
+        backgroundColor: theme.vars.palette.neutral.softBg,
+        visibility: visible ? 'visible' : 'hidden',
+        mb: 2,
+    });
+    
+
+    useEffect(() => {
+        setVisible(true);
+        const id= setTimeout(() => setVisible(false),5000);
```

Comment:
I think we should increase the timeout to 10 or 15 seconds. When the text came up, by the time I noticed and read it, it disappeared (the first few times). It might be better to leave it on there a little longer.

Reply:
ok. I will do that.